app/src/main/java/com/internship/move/feature/authentication/ForgotPasswordFragment.kt

```diff
@@ -25,8 +24,12 @@ class ForgotPasswordFragment : Fragment(R.layout.fragment_forgot_password) {
         }
 
         binding.sendResetLinkBtn.setOnClickListener {
-            Toast.makeText(context, "Reset link sent!!", Toast.LENGTH_SHORT).show()
-            // TODO("Alert dialog")
+            ForgotPasswordDialog().show(parentFragmentManager, FORGOT_PASSWORD_DIALOG_TAG)
         }
     }
+
+    companion object {
+
```

Comment:
No need to leave an empty first line inside a companion object. If I suggested this in the past then I apologize, it was a slip